handler/list.go

```diff
@@ -0,0 +1,33 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/cloudradar-monitoring/plexus/api"
+)
+
+// ListSession godoc
+// @Summary Lists all active session.
+// @Tags session
+// @Produce  application/json
+// @Success 200 [{object}] api.Session
+// @Failure 400 {object} api.Error
+// @Failure 500 {object} api.Error
+// @Failure 502 {object} api.Error
+// @Router /session [get]
+func (h *Handler) ListSession(rw http.ResponseWriter, r *http.Request) {
+	sessions := make([]*api.ListSessionItem, 0, len(h.sessions))
```

Comment:
There can be concurrent access to the h.sessions map, thus, the h.lock must be obtained before accessing the sessions on the handler.